project-2/src/admin/components/manage-film/modal/index.jsx

```diff
@@ -0,0 +1,142 @@
+import { Modal, Form, Input, Button, message } from "antd";
+import { useDispatch, useSelector } from "react-redux";
+import { useEffect, useState } from "react";
+import { updateMovie } from "../../../../redux/thunks/movie.thunk";
+import "./styles.scss";
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 20 },
+};
+
+const validateMessages = {
+  required: "${label} không được để trống",
+};
+
+function ModalFilm(props) {
+  const isModalVisible = props.isModalVisible;
+  const handleCancel = props.handleCancel;
+  const movie = props.movie;
```

Comment:
```suggestion
function ModalFilm({ isModalVisible, handleCancel, movie }) {
```
Sử dụng destructuring để thay thế nhé